build.sh

```diff
@@ -146,7 +146,7 @@ build_coreclr()
 
     if [ $__SkipConfigure == 0 ]; then
         # if managed build is suported, then set __GenerateVersionSource to true
-        if [ $__SkipMSCorLib == 0 ]; then __GenerateVersionSource=true; fi
+        if [ $__isMSBuildOnNETCoreSupported == 1 ]; then __GenerateVersionSource=true; fi
```

Comment:
This forces generating the version when the __isMSBuildOnNETCoreSupported, making the generateversion command line option useless - when there is msbuild available, we override the option and when it is not available, passing in the option would just break the build.
So I'd recommend changing that to if the __isMSBuildOnNETCoreSupported is zero, set the __GenerateVersionSource to false. 
Btw it would be nice if you could change the true / false to 1 / 0 to make it the same as we do for other options.